libmachine/mcnutils/b2d.go

```diff
@@ -38,12 +44,16 @@ func getClient() *http.Client {
 }
 
 type B2dUtils struct {
-	storePath        string
-	isoFilename      string
-	commonIsoPath    string
-	imgCachePath     string
-	githubAPIBaseURL string
-	githubBaseURL    string
+	storePath     string
+	isoFilename   string
+	commonIsoPath string
+	imgCachePath  string
+	defaultURL    string
+
+	// offset and length of ISO volume ID
+	// cf. http://serverfault.com/questions/361474/is-there-a-way-to-change-a-iso-files-volume-id-from-the-command-line
+	volumeIDOffset int64
```

Comment:
@nathanleclaire Actually I did at first, but I added these fields for testing.
I want to set the offset as a smaller value in testing.